### DISCUSSION TITLE
How to show last item of X axis?

### DISCUSSION THREAD

**Discussion by romanslonov** [f5/unovis#467] [Q&A] [answered]
Hello, in my graph I need to always show last date on X axis. What's the best way to do that? I am using vue.

![CleanShot 2024-10-21 at 18 59 02@2x](https://github.com/user-attachments/assets/1e7be70c-fa75-4e41-9bbe-3cf863b7a5d5)

I tried to set tick values explicitly but it just disappears.

```
:tick-values="[data?.[0]?.[index], data?.[data.length - 1]?.[index]]"
```

**Comment by rokotyan** [comment, marked as answer]
Setting `tickValues` explicitly should do the job. Check out the docks here https://unovis.dev/docs/auxiliary/Axis#set-ticks-explicitly

**Reply by romanslonov** [reply]
Oh yeah, I read that before, I was wrong to provide the actual piece of data for some reason instead of just index. Now it works.

Btw, quick question if you don't mind. I want to specify the minimum value for Y axis from 0 to 1 if the values ​​in the data are less than 1, if more than 1 then rely on the data, in this case I need to manually calculate the value of `yDomain: [0, computedValue]` right? Or `yDomainMinConstraint` can help?

**Reply by rokotyan** [reply]
`yDomainMinConstraint` and `yDomainMaxConstraint` have been designed exactly for this use case. 

Try setting **`yDomainMaxConstraint`** to `[1, undefined]`

**Reply by romanslonov** [reply]
Thank you so much, it works. One more question regarding `tickValues`, if I provide it, responsive stops working? I mean if it's not provided it changes automatically on resizing the window.

**Reply by rokotyan** [reply]
Yes, when you provide `tickValues`, only them will be displayed
